src/foam/nanos/auth/Address.js

```diff
@@ -25,10 +25,16 @@ foam.CLASS({
     'translationService'
   ],
 
+  javaImports: [
+    'foam.util.SafetyUtil'
+  ],
+
   messages: [
     { name: 'CITY_REQUIRED', message: 'City required' },
     { name: 'COUNTRY_REQUIRED', message: 'Country required' },
+    { name: 'INVALID_COUNTRY', message: 'Invalid country' },
     { name: 'REGION_REQUIRED', message: 'Region required' },
+    { name: 'INVALID_REGION', message: 'Invalid region' },
```

Comment:
Can you add that the ISO code is expected here?
```suggestion
    { name: 'INVALID_REGION', message: 'Invalid region. Please provide valid ISO-3166-2 region.' },
```